templates/project/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "lib": ["ESNext"],
+    "module": "CommonJS",
+    "target": "ES2018",
```

Comment:
Replacing ES-version based target with Node-version based template import from TypeScript lib might be more convenient for backend

Reply:
The version before, it using `fastify-tsconfig` and it hide all the options from user.
Using the official template face the same problem.